src/hooks/multiple-context.js

```diff
@@ -0,0 +1,9 @@
+import { useContext } from 'react'
+import AuthContext from '../context/AuthContext'
+import ProfileContext from '../context/ProfileContext'
+
+export const useMultipleContexts = () => {
+   const authCtx = useContext(AuthContext)
+   const profileCtx = useContext(ProfileContext)
+   return { ...authCtx, ...profileCtx }
```

Comment:
Maybe it's just my personal preference, but I don't like this approach of combining all context values in one object because

1. If you have many contexts with many values, and unintentionally duplicate the value key in two contexts (eg both AuthContext and ProfileContext having a `user` prop) - then we'll get just one of those, override the other, and that can lead to some difficult to find bugs :)
2. It's more explicit to know where the values are coming from and what they are related to if you have `const {user} = useAuthContext()` instead of `const {user} = useMultipleContexts()`
3. Also, pls create hooks for each context so we can do `const ... = useAuthContext()` instead of `const ...  useContext(AuthContext)` :)

Here's an example of using multiple contexts in the wild :)

![image](https://user-images.githubusercontent.com/5387195/153015443-fc3b8932-bbaf-42be-a0e6-d03cef477643.png)